### PR TITLE
fix: Update correct bash file on Mac

### LINKED ISSUE #930: jbang app setup not handling bashrc properly on osx
see #928, summary is that osx with default terminal app wont run .bashrc but runs bash_profile so wont ever see the path jbang configures.

### PR DESCRIPTION
**Pull request by quintesse**
Fixes #930

### CODE REVIEW

File: src/main/java/dev/jbang/cli/App.java [hunk 1]
```diff
@@ -359,8 +359,14 @@ public static int setup(boolean withJava, boolean force, boolean chatty) {
 		} else {
 			if (force || needsSetup() || withJava) {
 				// Update shell startup scripts
-				Path bashRcFile = getHome().resolve(".bashrc");
-				changed = changeScript(binDir, jdkHome, bashRcFile) || changed;
+				if (Util.isMac()) {
+					Path bashFile = getHome().resolve(".bash_profile");
+					changed = changeScript(binDir, jdkHome, bashFile) || changed;
```
[the remark this answers is not in the document]
Comment (quintesse) [reply src/main/java/dev/jbang/cli/App.java:364]: I'm afraid I have trouble parsing your sentence, can you rephrase please? 😅

[the remark this answers is not in the document]
Comment (maxandersen) [reply src/main/java/dev/jbang/cli/App.java:364]: Damn autocorrect. 

Asking if i have a bash_profile but uses iterm2 then at what time would the new Settings be read ?

[the remark this answers is not in the document]
Comment (quintesse) [reply src/main/java/dev/jbang/cli/App.java:364]: Supposedly iterm2 reads bash_profile as well. But like I said, I don't have a Mac so I can't test any of this, I'm completely dependent on you guys giving me correct information :-)

So what I've implemented is what was mentioned here: https://github.com/jbangdev/jbang/issues/930#issuecomment-859596875

[the remark this answers is not in the document]
Comment (maxandersen) [reply src/main/java/dev/jbang/cli/App.java:364]: Ok. I'll try once I'm off the autobahn;)

**Comment by maxandersen** [comment]
this seem to work for me, I'm sure it can be additionally refined (i.e. check `.profile` exists) but lets start with this. 

more details: https://scriptingosx.com/2017/04/about-bash_profile-and-bashrc-on-macos/
